index.js

```diff
@@ -34,6 +34,17 @@ module.exports = function (obj, opts) {
 			var key = keys[i];
 			var val = x[key];
 
+			if (deep && Array.isArray(val)) {
+				var retArr = [];
+
+				for (var j = 0; j < val.length; j++) {
```

Comment:
Might be preferable to use `ret[key] = val.map(i => isPlainObj(i) ? sortKeys(i) : i` ?